File: modul-02/PRAK205-2410817310014-INDRA_SURYADILAGA.c
#include <stdio.h>
#include <math.h>

int main() {
    int A, B;
    scanf("%d", &A);
    scanf("%d", &B);

    int Sisi_Tinggi = A;
    int Sisi_Miring = B;
    int Sisi_Alas = sqrt(Sisi_Miring * Sisi_Miring - Sisi_Tinggi * Sisi_Tinggi);
    int Keliling = Sisi_Alas + Sisi_Tinggi + Sisi_Miring;
    int Luas = Sisi_Tinggi * Sisi_Alas /2;

    printf("Alas     = %d cm \n", Sisi_Alas);
    printf("Tinggi   = %d cm \n", Sisi_Tinggi);
    printf("Keliling = %d cm \n", Keliling );
    printf("Luas     = %d cm^2 \n", Luas );
    return 0;
}

File: modul-02/PRAK205-2410817310014-INDRA_SURYADILAGA.py
import math

A,B = map(float,input().split())
Sisi_Tinggi = int(A)
Sisi_Miring = int(B)
Sisi_Alas = int(math.sqrt(Sisi_Miring * Sisi_Miring - Sisi_Tinggi * Sisi_Tinggi))
Keliling = int(Sisi_Alas + Sisi_Tinggi + Sisi_Miring)
Luas = int(Sisi_Tinggi * Sisi_Alas / 2)

print(f"Alas = {Sisi_Alas} cm")
print(f"Tinggi = {Sisi_Tinggi} cm")
print(f"Keliling = {Keliling} cm")
print(f"Luas = {Luas} cm^2")